syft/frameworks/torch/torch_attributes.py

```diff
@@ -232,6 +234,16 @@ def get_native_torch_name(attr: str) -> str:
         native_func_name = ".".join(parts)
         return native_func_name
 
+    def is_inplace_method(self, method_name):
+        """
+        Says if a method is inplace or not by test if it ends by _ and is not a __xx__
```

Comment:
is this going to be called during runtime or during hooking? If during runtime - is there a way we can avoid the string manipulation?

Reply:
Especially if this method is going to be executed for _every_ command

Reply:
oh wait - actually i see what's going on.

Oddly I think this is a good place to use a try/catch for speed. That way we don't have to check whether method_name is in self.inplace_methods twice (needlessly)

Reply:
*gasp!!!*